CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.5.0 (31/12/2015)
+
+### Changes
+
+* The dependency to Guzzle is removed. We are now using PHP-HTTP to make us independent from any transport library. 
+* We are using PSR7 requests and responses
+* Introduced HttpFactory to create clients and responses
```

Comment:
You sure that this is necessary? IMO this is what PHP HTTP message factory/discovery does.